.env.example

```
@@ -0,0 +1,25 @@
+# ----- Database -----
+# ローカル開発: Docker PostgreSQL
+DATABASE_URL=postgresql://postgres:postgres@localhost:5432/reflect_forward
+
+# 本番環境: Supabase (実際の値に置き換える)
+# DATABASE_URL=postgresql://postgres:[PASSWORD]@[HOST]:5432/postgres
+
+# ----- JWT -----
+# 本番環境では必ず強力なランダム文字列に置き換えること
+# 生成例: openssl rand -base64 32
+JWT_SECRET=your-secret-key-change-in-production
+JWT_EXPIRES_IN=15m
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_EXPIRES_IN key should go before the JWT_SECRET key

(UnorderedKey)